bin/collator/src/parachain/service.rs

```diff
@@ -1377,3 +680,35 @@ fn warn_if_slow_hardware(hwbench: &sc_sysinfo::HwBench) {
         );
     }
 }
+
+/// Start a parachain node.
+pub async fn start_node(
+    parachain_config: Configuration,
+    polkadot_config: Configuration,
+    collator_options: CollatorOptions,
+    para_id: ParaId,
+    additional_config: AdditionalConfig,
+) -> sc_service::error::Result<(TaskManager, Arc<FullClient>)> {
+    match parachain_config.network.network_backend {
+        NetworkBackendType::Libp2p => {
+            start_node_impl::<sc_network::NetworkWorker<_, _>>(
+                parachain_config,
+                polkadot_config,
+                collator_options,
+                para_id,
+                additional_config,
+            )
+            .await
+        }
+        NetworkBackendType::Litep2p => {
+            start_node_impl::<sc_network::Litep2pNetworkBackend>(
+                parachain_config,
+                polkadot_config,
+                collator_options,
+                para_id,
+                additional_config,
+            )
+            .await
+        }
+    }
```

Comment:
I decided not to integrate this while doing the uplift because I saw an open issue ticket to make this whole dependency optional - some project(s) weren't happy with the inclusion of this into the build.

I don't have anything strongly against adding this though, just wrote it for info sharing.